BuildLoader.py

```diff
@@ -339,13 +340,24 @@ def update_fit_table (self):
 			if len(u_code_images) > 0:
 				raise Exception('  Insufficient uCode entries in FIT. Need %d more.' % len(u_code_images))
 
-		if self._board.ACM_SIZE > 0:
+			acm3_index = 0
+			if self._board.ACM_SIZE == 0:
+				acm3_index       = num_fit_entries
+				num_fit_entries     += 1
+
 			# ACM
```

Comment:
1) ACM3 is not related with UCODE, should these code be moved out of "if self._board.UCODE_SIZE > 0:"? 
2) why num_fit_entries + 1 if self._board.ACM_SIZE is zero?

Reply:
Agree